packages/e2e/example/test_driver/example_e2e_test.dart

```diff
@@ -7,6 +7,6 @@ Future<void> main() async {
   final FlutterDriver driver = await FlutterDriver.connect();
   final String result =
       await driver.requestData(null, timeout: const Duration(minutes: 1));
-  driver.close();
+  await driver.close();
```

Comment:
https://github.com/flutter/plugins/edit/master/packages/e2e/README.md#L50
could also be updated! thx

Reply:
Good catch, thanks! Done.